pkg/controller/utils/utils.go

```diff
@@ -21,6 +18,15 @@ import (
 	routev1 "github.com/openshift/api/route/v1"
 )
 
+// DockerConfigJSON type would be be used to unmarshal pull secrets
+type DockerConfigJSON struct {
```

Comment:
`DockerConfigJSON` and `DockerConfig` do not need to be public.

Reply:
+1